libs/template/presets/storefront/experience/pages/cart-page.ts

```diff
@@ -72,6 +72,7 @@ export const cartPage: ExperienceComponent = {
               ],
             },
             { type: 'oryx-checkout-link' },
+            { type: 'oryx-cart-coupon' },
```

Comment:
Please, add featureVersion that we should show it after or equal 1.3. Same things with other changes in experience data page